quantum/keymap_extras/sendstring_bepo.h

```diff
@@ -13,29 +13,85 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-/* Sendstring definitions for the bépo layout */
-#ifndef SENDSTRING_BEPO
```

Comment:
Honestly, this is one of the places that I think we shouldn't use `#pragma once`, so that detection can happen.

Reply:
What would be a common use case for it? You're probably right, but I can't think of one.

Reply:
Just thinking about it, yeah, you're right.